.gitignore

```diff
@@ -6,6 +6,8 @@ _output/*
 _output
 sshfile
 ansible/*.txt
+ansible/gpu-test-logs/
+ansible/results/
```

Comment:
Do you think we could redirect those under _output directory, which is already excluded?